src/fragments/lib/auth/flutter/signin_with_custom_flow/10_cli_setup.mdx

```diff
@@ -0,0 +1,195 @@
+### How Custom Auth Works
```

Comment:
Should this be shared across platforms? Nothing in here seems specific to flutter. Are there many differences between this and the iOS docs?